root/static/scripts/common/immutable-entities.js

```diff
@@ -0,0 +1,89 @@
+// This file is part of MusicBrainz, the open internet music database.
```

Comment:
I expect this file will completely replace root/static/scripts/common/entity.js once we no longer have any need for knockout or observables. Uses simple immutable data structures + functions instead of classes.